### PR TITLE
Use full names for all voices

### LINKED ISSUE #9: Use proper long names for all voice names
Shorthands are just ugly and often not really even helpful. We have enough space, we should use it. I think this one can still be pushed into v2.6 :)

### PR DESCRIPTION
**Pull request by mkruselj**
Adjust the UI width and column width to fit

Closes #9

### CODE REVIEW

File: source/midi_syxg50_data.jsfx-inc [hunk 1]
```diff
@@ -710,7 +710,7 @@ X057:120:000 GM2 SFX
 //-----------------------------------------------------------------------------
 
 //#entry number
-//param number (translated to sysex address in code), min, max, default, display table, type (0:normal, 1:bipolar), name
+//param number (translated to sysex address in code), min, max, default, display table, name
```
Comment (samlletas) [inline source/midi_syxg50_data.jsfx-inc]: I think this comment can be useful to future contributors, to let them know what the column does. Was this removed accidentally by any chance?

**Comment by samlletas** [comment]
Seriously, many thanks for taking the time to do this! It looks much better compared to shorthands :)

I pushed a commit to this PR to remove the numbers for increased spacing and changed _"Trumpet & Trombone Section"_ to _"Trumpet & Trombone"_ since I felt the name was a bit too long (it also matches XGworks). 

I think the new voice names fit the window a bit better this way. What do you think?

**Edit:** Added two commits with more adjustments/ideas (decreased the column size to make the 3 of them fit the window, renamed 2 more long voices and renamed "Classic" drum kit to "Symphony" to match the XG spec).

File: source/midi_syxg50_data.jsfx-inc (hunk 1 above)
Comment (mkruselj) [reply source/midi_syxg50_data.jsfx-inc]: Yeah looks like it was. Will revert it!

**Comment by mkruselj** [comment]
Hmmm I'm not liking the random shortening of Organ to Org and Guitar to Gtr in those two cases. But also this PR was missing a crucial change where I changed the width of the window to 750px, so it'd be a 750x750 square. Then it would work out. Modifying...


EDIT: No actually the width change was already there, but yeah I don't see why we shouldn't have column width at 200 px. Even in worst case scenario (Synth Effects category) it's looking perfectly fine IMO.

**Comment by samlletas** [comment]
Yeah in particular because of the Synth Effects category (I found it weird how the selection rectangle went out of bounds when selecting a voice from the 3rd column) so I started exploring the idea of decreasing the column width.

Another approach could be to make everything fit in 2 columns, I pushed some changes where I adjusted the window size & spacing and reverted back "Trumpet & Trombone Section" since it now fits well in two columns. Also made sliders a bit taller to compensate for the new window height.

What are your thoughts on this idea?

**Comment by mkruselj** [comment]
Hmmm 800 px might be a bit on the hefty side, heheh. I think 3 columns were fine, and your change to use column width instead of list width for the selection rectangle is a good one and probably the only thing that was needed, perhaps? :)

**Comment by samlletas** [comment]
Hmmm for some reason I'm not entirely convinced of how the selection rectangle on the 3rd column looks narrower😅, but it only happens in one category so I'm fine leaving it at 3 columns, will see if I can think of something later for v3, maybe making the column width adaptive based on voice count, or something else that doesn't need renaming voices or resizing.

I'll undo my changes and merge the PR, again many thanks for these name improvements!

**Pull request merged**